inclusive-design/wecount.inclusivedesign.ca
# Hide tools card

## Description
Hid homepage "Tools" card


## Steps to test

1. run `npm run dev`
2. make sure tools card is no longer on the homepage
3. make sure rest of site still works

**Expected behavior:** 

The site should work the same with the exemption of the tools card on the homepage. It should no longer appear.
## Additional information



## Related issues



resolves #143
